lib/template-loader.php

```diff
@@ -107,6 +107,42 @@ function gutenberg_find_template( $template_file ) {
 	if ( $template_query->have_posts() ) {
 		$template_posts            = $template_query->get_posts();
 		$_wp_current_template_post = array_shift( $template_posts );
+
+		// Build map of template slugs to their priority in the current hierarchy.
+		$slug_priorities = new stdClass();
+		foreach ( $slugs as $index => $slug ) {
+			$slug_priorities->{$slug} = $index;
```

Comment:
```suggestion
			$slug_priorities[ $slug ] = $index;
```

Reply:
We can probably change that loop to just `$slug_priorities = array_flip( $slugs );`